src/components/CountDown/CountDown.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { IProps, IState } from "./types";
+import styles from "./CountDown.module.scss";
+
+export default class CountDown extends React.PureComponent<IProps, IState> {
+  private countDownInterval: any = null;
+
+  static defaultProps: Partial<IProps> = {
+    deadline: new Date(2020, 10, 3, 9, 0, 0),
```

Comment:
Watch out. `new Date()` expects a zero-index month. Check this in the console and see what it outputs, you'll notice that it targets November.